### ISSUE TITLE
Missing dependency at README

### ISSUE THREAD

**Issue by icaropires** [mozilla-services/fx-sig-verify#76]
I've tried to install the development environment following the steps on README, but I've got an error about missing openssl:
```
SWIG/_m2crypto_wrap.c:3541:10: fatal error: openssl/err.h: No such file or directory
     #include <openssl/err.h>
              ^~~~~~~~~~~~~~~
```
And after installing `openssl-devel` (Fedora), I've got the following:
```
    SWIG/_m2crypto_wrap.c:3754:35: error: 'CRYPTO_NUM_LOCKS' undeclared here (not in a function); did you mean 'CRYPTO_UNLOCK'?
     static PyThread_type_lock lock_cs[CRYPTO_NUM_LOCKS];
                                       ^~~~~~~~~~~~~~~~
                                       CRYPTO_UNLOCK
```

This is about openssl-devel version, so I've had to install `compat-openssl10-devel` and that worked for me.

So might be a good idea include that on README

**Comment by hwine** [comment]
Good catch. I agree something should be changed, but I'm considering a different approach.

**Comment by hwine** [comment]
Closed by #78
